src/functionality.js

```diff
@@ -0,0 +1,92 @@
+export default class Tasks {
+  constructor() {
+    this.tasksList = [];
+  }
+
+  removeTask(index) {
+    this.tasksList.splice(index, 1);
+    for (let i = index; i < this.tasksList.length; i += 1) {
+      this.tasksList[i].index = i;
+    }
+    localStorage.setItem('tasks', JSON.stringify(this.tasksList));
+    this.updateDisplay();
+  }
```

Comment:
- [ ] Kindly make sure when we click on the delete icon user can be able to remove the task on the list.

    ![chrome-capture-2022-10-12](https://user-images.githubusercontent.com/87197412/201469867-28dcbfe7-4dbd-4634-8cb1-a6af889a67bd.gif)

Reply:
Made changes to the delete feature